cmd/backend/pull_config/pull-config.go

```diff
@@ -0,0 +1,236 @@
+
+package pull_config
+
+/*
+$ go run main.go backend pull-config
+*/
+
+import (
+	"os"
+	"strings"
+    "log"
+    "fmt"
+    
+	"github.com/spf13/cobra"
+    "github.com/DSGT-DLP/Deep-Learning-Playground/cli/cmd/backend"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm" // For Parameter Store
+)
+
+var pullConfigCmd = &cobra.Command{
+	Use:   "pull-config",
+	Short: "Pulls values from parameter store/secrets manager and writes them to .env files",
+	Long: `This command pulls values from parameter store/secrets manager and writes them to .env files in /training and /frontend.`,
+    Args:  cobra.ExactArgs(0),
+	Run: func(cmd *cobra.Command, args []string) {
+		sess, err := session.NewSession(&aws.Config{
+			Region: aws.String("us-west-2")},
+		)
+
+		if err != nil {
+			log.Fatal("Error creating AWS session: ", err)
+		}
+
+		ssmClient := ssm.New(sess)
+
+		parameterNames := []string{"parameter-name"}
+
+		for _, paramName := range parameterNames {
+			result, err := ssmClient.GetParameter(&ssm.GetParameterInput{
+				Name:           aws.String(paramName),
+				WithDecryption: aws.Bool(true),
+			})
+
+			if err != nil {
+				log.Fatal("Error getting parameter: ", err)
+			}
+
+			writeToEnvFile(paramName, *result.Parameter.Value, backend.BackendDir)
```

Comment:
will `serverless/` directory also need parameter store values or not really?

Reply:
lemme circle back to you on this one.

Reply:
You'll ping @dwu359 on this?

Reply:
that was my plan. @dwu359 your thoughts on this?

Reply:
@dwu359 your thoughts on this? we can also talk about this on Sunday.

Reply:
sounds good. can you also ping daniel in the #backend channel of DLP discord @codingwithsurya

Reply:
he said yea . resolving this convo now.